pkg/lint/rules/chartfile_test.go

```diff
@@ -30,16 +30,19 @@ import (
 
 const (
 	badChartDir  = "testdata/badchartfile"
+	badNameChartDir = "testdata/badnamechart"
 	goodChartDir = "testdata/goodone"
 )
 
 var (
 	badChartFilePath         = filepath.Join(badChartDir, "Chart.yaml")
+	badNameChartFilePath     = filepath.Join(badNameChartDir, "Chart.yaml")
```

Comment:
I think you intended the variable to be named `badChartNameFilePath` instead. It doesn't look like `badNameChartFilePath` is used anywhere in this file. Line 45 is `badChartNameFilePath`